labs/spring_2016_update.md

```diff
@@ -0,0 +1,30 @@
+Subject: Hack Club Semester 2 Update
+
+Hi everyone!
+
+We've learned and failed a tremendous amount this past semester. All of you
+have been incredibly supportive and both of us feel so privileged to have the
+opportunity to work with each and every one of you.
+
+In the next two months, Hack Club is focusing on growth. Our goal is to start
+50 clubs and increase our weekly attendance by 750 students by the end of
+February. As a result of this, we're focusing on rewriting our early workshops
+and standardizing the beginning of clubs ([click here][club_guide] for an
+example of what we're working on). This means that we won't be building
+curriculum for clubs like yours at the pace we did last semester and we may
+not have new workshops in time for all of your meetings while we're still
+early into this semester.
+
+
+Your work this past semester has led us to where we are today and we'll only
+be able to continue the movement with your continued involvement. As we
+onboard new clubs, the best way for you to contribute is to welcome and
+support new club leaders. Make the Slack feel like it's their home. We're
+looking to pair existing clubs with new clubs to start the mentorship early.
+If you are interested in being a mentor to new clubs, please reply directly to
+this email to let us know.
+
+Let's make 2016 amazing!
```

Comment:
Should the end of this line have two spaces at the end to force the Markdown rendered to add a newline? See the "break" part of https://daringfireball.net/projects/markdown/syntax#p.